src/plugins/intel_gpu/src/graph/impls/cm/pa_single_token.cm

```diff
@@ -450,7 +450,7 @@ extern "C" _GENX_MAIN_ void KERNEL_NAME(
             #endif
             cm_svm_block_write<float, REG_N>((svmptr_t)(output + o_offset + k),cur_O_f32.format<float>());
         }
-        uint lse_offset = seq_idx * KV_HEADS_NUM * kv_partition_num + (head_num_idx + qi) * kv_partition_num + wg_thread_id;
+        uint lse_offset = seq_idx * HEADS_NUM * kv_partition_num + (head_num_idx + qi) * kv_partition_num + wg_thread_id;
```

Comment:
Good finding, It should work for multi-sequences.
We didn't notice this issue for current single-sequence.